index.js

```diff
@@ -887,6 +887,19 @@ var QUERIES = [
       return result
     }
   },
+  {
+    regexp: /^supports\s+(\w+)$/,
```

Comment:
`-` is not part of `\w` but widely used in `feat` code

Reply:
Added support for dashes on features names.